Expand Difficulty Increase mod articles (#12629)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Accuracy_Challenge/en.md

*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Accuracy Challenge** mod allows you to set a minimum accuracy that you can't exceed to avoid a failure.

Reviewer: ```suggestion
The **Accuracy Challenge** mod allows the player to set a minimum accuracy that they must maintain to avoid a failure.
```

## Customisation

![Accuracy Challenge customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/AC.jpg)

> - **Minimum accuracy** (60%-99%, default 90%): Trigger a failure if your accuracy goes below this value.
> - **Accuracy mode** (default: Maximum achievable): The mode of accuracy that will trigger failure.
>   - **Maximum achievable**: Fail if you can’t achieve an accuracy higher than the value when the map ends.

Reviewer: ```suggestion
- **Minimum accuracy** (60%-99%, default: 90%): Trigger a failure if accuracy goes below this value.
- **Accuracy mode** (default: Maximum achievable): The mode of accuracy that will trigger failure.
  - **Maximum achievable**: Fail if accuracy doesn't exceed this value when the map ends.
```

>   - **Standard**: Fail if the accuracy strictly goes below the value.

Reviewer: ```suggestion
  - **Standard**: Fail if the accuracy goes strictly below the value.
```

> - Restart on fail (default disabled): Restart the map after the accuracy goes below the minimum value indicated.

Reviewer: ```suggestion
- Restart on fail (default: disabled): Restart the map when the accuracy goes below the minimum value specified.
```
